### PR TITLE
Bug 1893889: Update pf dependencies to 2021-1 to fix a117 issues with disabled dropdown items

### PR DESCRIPTION
**Pull request by jcaianirh**
Fixes: https://bugzilla.redhat.com/show_bug.cgi?id=1893889

Updates pf dependencies to release 2021.1

### CODE REVIEW

File: frontend/public/components/graphs/tooltip.tsx [hunk 1]
```diff
@@ -43,7 +43,8 @@ export const ChartLegendTooltipContent: React.FunctionComponent<ChartLegendToolt
   theme = getTheme(themeColor, themeVariant),
   ...rest
 }) => {
-  const pointerLength = theme && theme.tooltip ? theme.tooltip.pointerLength : 10;
+  const pointerLength =
+    theme && theme.tooltip ? Helpers.evaluateProp(theme.tooltip.pointerLength) : 10;
```
Comment (spadgett) [inline frontend/public/components/graphs/tooltip.tsx:47]:
```suggestion
    theme?.tooltip ? Helpers.evaluateProp(theme.tooltip.pointerLength) : 10;
```

File: frontend/public/components/graphs/tooltip.tsx [hunk 2]
```diff
@@ -231,7 +232,8 @@ export const ChartLegendTooltip: React.FunctionComponent<ChartLegendTooltipProps
   stack,
   ...rest
 }) => {
-  const pointerLength = theme && theme.tooltip ? theme.tooltip.pointerLength : 10;
+  const pointerLength =
+    theme && theme.tooltip ? Helpers.evaluateProp(theme.tooltip.pointerLength) : 10;
```
Comment (spadgett) [inline frontend/public/components/graphs/tooltip.tsx:236]:
```suggestion
    theme?.tooltip ? Helpers.evaluateProp(theme.tooltip.pointerLength) : 10;
```

**Comment by spadgett** [comment]
/assign @rhamilto 

@rhamilto Can you help validate these changes?

@jcaianirh Please squash, thanks!

**Comment by rhamilto** [comment]
> /assign @rhamilto
> 
> @rhamilto Can you help validate these changes?

Pulled PR and tested.  Everything looks good to me.  Just need @jcaianirh to squash.

**Comment by jcaianirh** [comment]
Looks like there is an issue with the table action column which outputs an empty scope attribute. Added a pf issue here: https://github.com/patternfly/patternfly-react/issues/5373 which is causing the cypress a11y tests to fail

**Comment by jcaianirh** [comment]
/hold
until above issue is resolved

**Comment by jcaianirh** [comment]
Updated with latest prerelease packages for 2020.17 which addresses fix for broken a11y cypress test. Keeping hold until pf officially releases (probably pending no issues 1/26/2021)

**Comment by jcaianirh** [comment]
@rhamilto new pf dependencies added for this pr if you wanna take another look

**Review by rhamilto — APPROVED** [review]
/lgtm

**Pull request merged**